src/pkg/config/manager.go

```diff
@@ -34,6 +35,20 @@ type CfgManager struct {
 
 var validateRules = []validate.Rule{
 	&validate.LdapGroupValidateRule{},
+// CheckModifiable check if the configure item can be modified
+func (c *CfgManager) CheckModifiable(ctx context.Context, cfgs map[string]interface{}, keys ...string) (failed []string) {
```

Comment:
I don't understand the logic here, it checks the input cfg and the value in DB, it fails if there's any difference?

Could you add more comment to explain?

Reply:
Refactor to AuthModeUpdated